compiler/packages/babel-plugin-react-compiler/src/HIR/visitors.ts

```diff
@@ -193,7 +193,7 @@ export function* eachInstructionValueOperand(
     }
     case 'ObjectMethod':
     case 'FunctionExpression': {
-      yield* instrValue.loweredFunc.dependencies;
+      yield* instrValue.loweredFunc.func.context;
```

Comment:
This is a little different than all other visited `places`, as function contexts are currently lvalues (i.e. they directly reference the relevant identifier instead of lowering to an artificial `LoadLocal` / `LoadContext`).